bin/run-browser-test-env

```diff
@@ -47,6 +54,9 @@ Options:
 
 --azure Use the Azure emulator (Azurite) for simulating cloud storage for file
         upload and download. Also use localstack, since we need that for SES.
+
+--local Set BASE_URL to http://localhost:9999 to enable running
```

Comment:
Is it possible for `--local` to be `--local_mode` to match? Or do they necessarily have to be different?

Reply:
No, they could be the same.  I'm not sure why I chose `local_mode` for the var name elsewhere :)

Reply:
Would you make that change? Unless you agree with my suggestion below.